HotelBooking/Views/reserved_rooms_admin_view.py

```diff
@@ -0,0 +1,272 @@
+from HotelBooking.Controllers.reservation_controller import ReservationController
+from HotelBooking.Controllers.room_controller import RoomController
+from HotelBooking.Controllers.bill_controller import BillController
+from HotelBooking.Views.view import View
+from typing import Tuple, List
+from PyInquirer import prompt
+from HotelBooking.Models.room import ROOM_TYPE
+PROMPT_KEY = {
+    "OPERATIONS": 'operations',
+    "RESERVE": 'reserve',
+    "CANCEL": 'cancel',
+    "START_DATE": "start_date",
+    "DURATION": "duration",
+    "BACK": "back",
+    "FINAL_CHECK": "final_check",
+    "CUSTOMER": "customer"
+}
+
+PROMPTS = {
+    'operations': [{
+        'type': 'list',
+        'message': "Reserve/Cancel a room for a guest",
+        'name': 'operations',
+        'choices': [
+        ]
+    }],
+    'reserve': [{
+        'type': 'list',
+        'message': "What type of room?",
+        'name': 'reserve',
+        'choices': [
+        ],
+        'filter': lambda choice: choice.split(" ")[0].upper(),
+    }],
+    'cancel': [{
+        'type': 'list',
+        'message': "Which one to cancel?",
+        'name': 'cancel',
+        'choices': [
+        ]
+    }],
+    "customer": [{
+        'type': 'input',
+        'message': "Enter the customer's ID",
+        'name': 'customer',
+    }],
+    "start_date": [{
+        'type': 'input',
+        'message': "Enter the start date",
+        'name': 'start_date',
+    }],
+    "duration": [{
+        'type': 'input',
+        'message': "Enter the amount of date you want to reserve",
```

Comment:
Change to "Enter the number of days you want to reserve"

Reply:
fixed